tensorflow_lite_support/web/tflite_model_runner/demo/src/index.html

```diff
@@ -65,7 +65,6 @@ <h1>TFLite Web API Model Runner Demo</h1>
     <img src="wine.jpeg" alt="wine"/>
     <div class="result"></div>
   </div>
-  <script src="./webnn-polyfill/webnn-polyfill.js"></script>
```

Comment:
Don't remove webnn-polyfill, we should update it once it supports latest sync API.